recipes/dav1d/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "dav1d" %}
+{% set version = "1.0.0" %}
+# library version is different from project version
+# Check 'meson.build' upstream in every release!
+{% set so_name = "6.6.0" %}
+{% set so_name_major = "6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://code.videolan.org/videolan/dav1d/-/archive/{{ version }}/dav1d-{{ version }}.tar.gz
+  sha256: 047b8229511a82b5718a1d34c86c067b078efd02f602986d2ed09b23182ec136
+
+build:
+  number: 0
+  script:
+    - set -ex
+    - >
+        meson setup _build
+        ${MESON_ARGS:---prefix=${PREFIX} --libdir=lib}  # [unix]
+        --prefix=%LIBRARY_PREFIX%  # [win]
```

Comment:
Nice trick!

Reply:
Saw it in another feedstock using Meson!